hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report_views.xml

```diff
@@ -49,6 +49,37 @@
         <field name="view_mode">pivot,graph</field>
     </record>
 
+    <record id="hr_attendance_theoretical_view_employee_pivot" model="ir.ui.view">
+        <field name="model">hr.attendance.theoretical.time.report</field>
+        <field name="sequence">99</field>
+        <field name="arch" type="xml">
+            <pivot disable_linking="1">
+                <field name="date" type="row"/>
+                <field name="employee_id" type="col"/>
+                <field name="worked_hours" type="measure" widget="float_time"/>
+                <field name="theoretical_hours" type="measure" widget="float_time"/>
+                <field name="difference" type="measure" widget="float_time"/>
+            </pivot>
+        </field>
+    </record>
+
+    <record id="hr_attendance_theoretical_employee_action" model="ir.actions.act_window">
```

Comment:
The same for the action. If you need something different (like the context), use a method in the model reading the act_window and modifying it accordingly

Reply:
I've used a new action to be able to set my special view for the action - also i did have to remove the default context.
It would be possible to do this in a model function, but because it does also modify the used view - i think it is better to create a new action

Reply:
Yeah, but following my previous comment logic, I still don't think it's needed.